packages/tkg-clusterclass/bundle/config/packageinstalls/tkg-infra-clusterclass-pi.yaml

```
@@ -72,3 +73,177 @@ metadata:
 type: Opaque
 stringData:
   values.yaml: #@ yaml.encode(data.values.clusterclassInfraPackageValues.configValues)
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tkg-clusterclass-infra-overlays
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-rule.0: "upsert after upserting tkg-infra-clusterclass-packageinstall/serviceaccount"
+    kapp.k14s.io/change-rule.1: "delete before deleting tkg-infra-clusterclass-packageinstall/serviceaccount"
+type: Opaque
+stringData:
+  kcp-empty-array-ovleray.yaml: |
```

will this cause an error somewhere?
```suggestion
  kcp-empty-array-overlay.yaml: |
```

---

Nope, renaming is fine. Carvel will use all files of the config map 👍